tests/schemes/test_thrift.py

```diff
@@ -1162,3 +1163,80 @@ def testException(request):
     )
 
     assert 1 == res.status_code
+
+
+##############################################################################
+# Calling registered functions directly should be equivalent to calling them
+# as if they weren't registered at all
+
+
+def body(**kwargs):
```

Comment:
guessing we'll have to do work to make the request easier to inject into the endpoint?